tslint.json

```diff
@@ -49,10 +78,14 @@
         "no-any": false,
         "no-arg": true,
         "no-backbone-get-set-outside-model": true,
-        "no-banned-terms": true,
+        "no-banned-terms": false,
```

Comment:
Among the "terms" that are banned is `arguments` which tslint (perhaps erroneously) flagged as banned when found in src/shapes.ts:
```js
export interface Manifest {
    ...
    runtime: {
        arguments?: string;
        fallbackVersion?: string;
        forceLatest?: boolean;
        futureVersion?: string;
        version: string;
    };
```
And since this is already in our API, my only choice was to turn it off.

I say "perhaps erroneously" because it only objected in the above declaration, but didn't object to where it was actually used (dereferenced), such as in process_tracker.js:208:
```js
var exeArgs = options.arguments || aliasJsonObject.args || '';
```

Also, it would not allow us to use `type` which we use a lot.